test/e2e/app-dir/actions/app/error-handling/page.js

```diff
@@ -18,6 +21,15 @@ export default function Page() {
       >
         Submit
       </button>
+
+      <button
+        id="submit-transition"
+        onClick={async () => {
+          startTransition(() => action())
```

Comment:
is startTransition + action supposed to error? Does normal case like throwing error from action is also handled?

Reply:
The transition is so we can ensure the error is caught by an error boundary ([ref](https://react.dev/reference/react/useTransition#displaying-an-error-to-users-with-error-boundary)) since the error is thrown in an event handler. The other case is indeed handled and tested elsewhere.